syntax/rust.vim

```diff
@@ -32,6 +32,7 @@ syn keyword   rustKeyword     fn nextgroup=rustFuncName skipwhite skipempty
 syn keyword   rustKeyword     in impl let
 syn keyword   rustKeyword     macro
 syn keyword   rustKeyword     pub nextgroup=rustPubScope skipwhite skipempty
+syn keyword   rustKeyword     crate skipwhite skipempty
```

Comment:
`skipwhite skipempty` is only needed (or desirable) if `nextgroup` is specified. Also note that all these keywords are in alphabetical order so far, so putting this one with or after `continue` (I have no particular preference) would be good.

Reply:
Should be fixed now.